.circleci/config.yml

```
@@ -81,10 +105,26 @@ jobs:
       - store_test_results:
           path: tmp/results
 
+ignore_main:
```

I think you need to name this reference in order to splat it below?
```suggestion
ignore_main: &ignore_main
```

---

Thanks, was just looking at that and was rather confused 😅